src/sentry/api/endpoints/organization_sdk_deprecations.py

```diff
@@ -0,0 +1,138 @@
+from collections import defaultdict
+from typing import DefaultDict, TypedDict
+
+import sentry_sdk
+from packaging.version import InvalidVersion, Version
+from packaging.version import parse as parse_version
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.models.project import Project
+from sentry.models.projectsdk import EventType, ProjectSDK
+
+
+class SDKDeprecationsSerializer(serializers.Serializer):
+    event_type = serializers.ChoiceField(
+        choices=("profile",),
+        required=True,
+    )
+
+
+class SDKDeprecation(TypedDict):
+    projectId: str
+    minimumVersion: str
+    sdkName: str
+    sdkVersion: str
+
+
+class SDKDeprecationsResult(TypedDict):
+    data: list[SDKDeprecation]
+
+
+@region_silo_endpoint
+class OrganizationSdkDeprecationsEndpoint(OrganizationEndpoint):
+    owner = ApiOwner.PROFILING
+
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+    }
+
+    def get(self, request: Request, organization) -> Response:
+        serializer = SDKDeprecationsSerializer(data=request.GET)
```

Comment:
i wonder if from an API design standpoint it makes sense to just return all deprecation notices for the project, there can only ever be so many, right?

Reply:
To do this, we would have to return all sdks versions used by each project. Which would mean the endpoint should use a list of all used sdks.